MediaRecorder.bs

```diff
@@ -101,8 +102,8 @@ interface MediaRecorder : EventTarget {
         "False">&#10008;</span></td>
         <td class="prmOptFalse"><span role="img" aria-label=
         "False">&#10008;</span></td>
-        <td class="prmDesc">The {{MediaStream}} to be recorded. This will
-        be the value of the {{MediaRecorder/stream}} attribute.</td>
+        <td class="prmDesc">The {{MediaStream}} to be recorded. This
+        can be changed later using the {{MediaRecorder/stream}} attribute.</td>
```

Comment:
s/using/by setting/ ?

Is it worth noting that operations done on the stream attribute does not affect the tracks recorded (and vice versa, operations done affecting the recorded tracks does not reflect in the stream attribute's track set)?